src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.ts

```diff
@@ -90,98 +85,54 @@ export class WorkItemLinkComponent implements OnInit, OnChanges {
         
     }
 
-    onSelectRelation(relation: LinkType): void{
+    onSelectRelation(relation: any): void{
```

Comment:
Can you use a type for that? Maybe a number or string? Just asking if it is possible.